package.json

```diff
@@ -36,14 +36,14 @@
         "graphql": "^15.4.0",
         "graphql-tag": "^2.11.0",
         "gud": "^1.0.0",
-        "i18next": "^19.0.3",
-        "i18next-xhr-backend": "^3.2.2",
+        "i18next": "^23.16.8",
+        "i18next-http-backend": "^3.0.2",
```

Comment:
I see that xhr is used quite a bit, will we need to update these projects as well?: https://github.com/search?q=org%3AJahia+%22i18next-xhr-backend%22&type=code

Reply:
Thanks for bringing it up. For the most part it's mostly just referred in the standard webpack.shared.js which is mostly ignored unless it's also part of the package.json dependency for the module. 

For the others https://github.com/search?q=org%3AJahia+%22i18next-xhr-backend%22+path%3Apackage.json&type=code, they should still be able to use and share this module if needed. e.g. I tested with AS-ui module which has it in the package.json and still able to use translations.

I can create a follow-up clean up task for those modules with the old xhr dependency (we don't have much outside of support modules) but not necessary for us to merge this PR i.e. should have no conflicts/issues